docs/source/scheduling.rst

```diff
@@ -20,13 +19,13 @@ but with different performance characteristics.
 
 Dask has two families of task schedulers:
 
-1.  **Single machine scheduler**: This scheduler provides basic features on a
+1.  **Single machine scheduler**: this scheduler provides basic features on a
     local process or thread pool.  This scheduler was made first and is the
-    default.  It is simple and cheap to use.  It can only be used on a single
-    machine and does not scale.
-2.  **Distributed scheduler**: This scheduler is more sophisticated, offers
+    default.  It is simple and cheap to use, altough it can only be used on 
+    a single machine and does not scale
+2.  **Distributed scheduler**: this scheduler is more sophisticated, offers
     more features, but also requires a bit more effort to set up.  It can
-    run locally or distributed across a cluster.
```

Comment:
I think that we should keep the period here.  This is a complete sentence and is not in a place where we care about minimizing visual space (like a title page)

Reply:
I see your point. For short descriptions it's fine not ending with a dot, and it makes sense using periods for *full* sentences, but what would be the case where it is hard to distinguish between these two (i.e., when its a long description but not quite a sentence)? So far I've been using the simple rule of not using periods in lists (like in #4020)

Would the rule of thumb be no ending dots for all short descriptions and dots for more than one sentence per item in a list? or avoid using ending dots completely in lists?

What's your thoughts about this?

Reply:
> but what would be the case where it is hard to distinguish between these two

No strong thoughts.  Defaulting to no-dots seems fine if you prefer.

Reply:
Lets not use dots as default for now and latter change it if it seems more appropriate